SLHCUpgradeSimulations/Configuration/python/fixMissingUpgradeGTPayloads.py

```diff
@@ -19,3 +19,33 @@ def fixRPCConditions(process):
     return process
 
 
+def fixDTAlignmentConditions(process):
```

Comment:
are these functions needed?
Which GT are you using for testing?

Neither of these functions are used in this PR.